src/main/java/frc/robot/subsystems/FourBarArm.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.math.controller.PIDController;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxAlternateEncoder;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.ArmConstants;
+
+public class FourBarArm extends SubsystemBase {
+  private final CANSparkMax m_motor;
+  private final PIDController m_pid;
+  private final RelativeEncoder m_encoder;
+
+  public FourBarArm() {
+    m_motor = new CANSparkMax(ArmConstants.motorID, MotorType.kBrushless);
+    m_motor.setIdleMode(IdleMode.kBrake);
+    m_encoder = m_motor.getAlternateEncoder(SparkMaxAlternateEncoder.Type.kQuadrature, 8192);
+    m_pid = new PIDController(ArmConstants.kP, ArmConstants.kI, ArmConstants.kD);
+  }
+
```

Comment:
Add an @Override periodic method to run the pid if the subsystem is enabled here